simple.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+let input = FileHandle.standardInput
+
+if let text = String(data: input.readDataToEndOfFile(), encoding: .utf8) {
```

Comment:
I think this line violates the constraint mentioned the _Problem Statements and Constrains_ section of the [blog post](https://benhoyt.com/writings/count-words/). Solutions aren't supposed to load the entire file into memory. Maximum allowed is a 64KB chunk.

Reply:
I just skimmed the blog post, changed the code accordingly.

Reply:
👍